docs/docs/system-administration/importing-and-exporting/organisations.md

```diff
@@ -33,150 +36,213 @@ We **will** export the following entities:
-Organisations.
+<summary>Kubernetes</summary>
 
-This process does **not** support importing data into an existing Organisation.
+To run an interactive shell inside an existing API container, use `kubectl exec` replacing `YOUR_API_SERVICE` with the 
+name of your Flagsmith API Kubernetes service:
+
+```
+kubectl exec -it service/YOUR_API_SERVICE --container flagsmith-api -- sh
+```
+
+To find your Flagsmith API Kubernetes service, you can use `kubectl get services`:
+
+```
+kubectl get services --selector app.kubernetes.io/component=api
+```
+
+Putting these two commands together, this one-liner will give you an interactive API shell:
+
+```
+kubectl exec -it  $(kubectl get service --selector app.kubernetes.io/component=api --output name) --container flagsmith-api -- sh
+```
+
+</details>
+
+<details>
+
+<summary>Docker Compose</summary>
+
+Use `docker compose exec` to get an interactive shell inside your API container, replacing `flagsmith` with your 
+Flagsmith API service name from your Compose definition:
+
+```
+docker compose exec -it flagsmith sh
+```
+
+</details>
+
+<details>
+
+<summary>SSH and local environments</summary>
+
+If you have a shell inside a Flagsmith environment, check that you can run `python manage.py`. In containers running 
+Flagsmith images, this file is located in the `/app` directory:
+
+```
+python /app/manage.py health_check
+```
+
+</details>
 
 ## Exporting
 
-The export process involves running a command from a terminal window. This must either be run from a running container
-in your self hosted deployment or, alternatively, you can run a separate container that can connect to the same database
-as your deployed fleet of flagsmith instances. To run the command, you'll also need to find the id of your organisation.
-You can do this through the django admin interface. Information about accessing the admin interface can be found
-[here](/deployment/configuration/django-admin.md). Once you've obtained access to the admin interface, if you browse to
-the 'Organisations' menu item on the left, you should see something along the lines of the following:
+To export your Flagsmith organisation, you need to know its ID. To find an organisation's ID, use one of the 
+following methods:
 
-![Image](/img/organisations-admin.png)
+* From the Flagsmith dashboard, click your organisation name in the top left. The organisation ID is displayed in 
+  the URL bar: `https://flagsmith.example.com/organisation/YOUR_ORGANISATION_ID/...`.
+* From [Django Admin](/deployment/configuration/django-admin), browse to the Organisations section in the sidebar. 
+  Here you can see all of your organisations and their IDs. 
+* If you have an Admin API key, call the
+  [List Organisations API endpoint](https://api.flagsmith.com/api/v1/docs/#/api/api_v1_organisations_list). This 
+  returns all the organisations that your API key is a member of.
```

Comment:
Technically you can use the 'show JSON references' here too, but maybe that's just more complicated to explain here than needs be?

Reply:
It's 2 more clicks than looking at the URL bar (open Org Settings, then expand the JSON data), not everyone has it enabled, and users that don't want it will need to disable it again. We also display the org ID in the Billing section but this is not shown onprem:

<img width="1360" alt="image" src="https://github.com/user-attachments/assets/f3a128a1-f8a6-4bda-b6ae-b0aa403972e2" />

IMO we should just display the org ID somewhere easier to find, which I didn't want to think about for this PR.

Reply:
Yeah, agreed. I think in the past I've seen scenarios where like clicking on the organisation name in the top left copies the ID to your clipboard or something. Agreed that it's not in the scope of this PR, and I'm happy to just suggest the URL approach.